algorithms/kernel/svm/oneapi/cl_kernels/svm_kernels.cl

```diff
@@ -0,0 +1,102 @@
+/* file: svm_kernels.cl */
+/*******************************************************************************
+* Copyright 2020 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+/*
+//++
+//  Implementation of SVM kernels.
+//--
+*/
+
+#ifndef __SVM_KERNELS_CL__
+#define __SVM_KERNELS_CL__
+
+#include <string.h>
+
+#define DECLARE_SOURCE_DAAL(name, src) static const char *(name) = #src;
+
+DECLARE_SOURCE_DAAL(
+    clKernelSVM,
+
+    __kernel void initGradient(const __global algorithmFPType * const y, __global algorithmFPType * grad) {
```

Comment:
It`s not clear outside this file how you init the gradients. Probably, rename this function, e.g. `initGradientByYFlip`?